tests/fixtures-compact.md

```diff
@@ -0,0 +1,137 @@
+simple
+.
+a | bb
+| - | -
+1 | 2
+.
+| a | bb |
+| -- | -- |
+| 1 | 2 |
+.
+
+empty headers
+.
+|  | |
+|- | -
+|1 | 2
+.
+|  |  |
+| -- | -- |
+| 1 | 2 |
+.
+
+no body
+.
+|  | |
+|- | -
+.
+|  |  |
+| -- | -- |
+.
+
+alignment
+.
+a | b | c
+:- | -: | :-:
+1 | 2 | 3
+xxxxxx | yyyyyy | zzzzzz
+.
+| a | b | c |
+| :- | -: | :: |
+| 1 | 2 | 3 |
+| xxxxxx | yyyyyy | zzzzzz |
```

Comment:
This formatting is incorrect because two dashes are expected:

alignment

a | b | c
:- | -: | :-:
1 | 2 | 3
xxxxxx | yyyyyy | zzzzzz

| a | b | c |
| :- | -: | :: |
| 1 | 2 | 3 |
| xxxxxx | yyyyyy | zzzzzz |